crawl-ref/source/util/gen_util.py

```diff
@@ -0,0 +1,83 @@
+#!/usr/bin/env python
+"""Utility functions for python scripts converting yaml to C++ headers."""
+
+# from __future__ import print_function
+
+# import argparse
+import os
+import sys
+import traceback
+import collections
+import re
+
+# import yaml  # pip install pyyaml
+
```

Comment:
Currently some of these import statements are not required in this module, but there's a fair chance that if datification of data files continues to take place over coming versions, they may be necessary in this module. As a result, I've left them commented out. Would you prefer them to be removed entirely, or uncommented, or is commenting them out ok?